scripts/cmake/vcpkg_configure_make.cmake

```
@@ -278,8 +288,10 @@ function(vcpkg_configure_make)
     # Backup environment variables
     # CCAS CC C CPP CXX FC FF GC LD LF LIBTOOL OBJC OBJCXX R UPC Y 
     set(cm_FLAGS AS CCAS CC C CPP CXX FC FF GC LD LF LIBTOOL OBJC OBJXX R UPC Y RC)
+    set(cm_TOOLS CC CXX AR AS LD RANLIB STRIP)
     list(TRANSFORM cm_FLAGS APPEND "FLAGS")
     vcpkg_backup_env_variables(VARS ${cm_FLAGS})
+    vcpkg_backup_env_variables(VARS ${cm_TOOLS})

```

```suggestion
    vcpkg_backup_env_variables(VARS ${cm_FLAGS} ${cm_TOOLS})
```

no reason to not do it in one call

---

accepted